libs/community/tests/integration_tests/test_bigquery_vector_search.py

```diff
@@ -106,3 +113,27 @@ def test_get_doc_by_filter(self, store: BigQueryVectorSearch) -> None:
         assert "fruit" in kinds
         assert "treat" not in kinds
         assert "planet" not in kinds
+
+    @pytest.mark.skip(reason="investigating")
```

Comment:
why is it marked as skipped? is this new test failing?

Reply:
Hi,
My bad I edited the wrong test file. It was the tets file for the outdated BigQueryVectorSearch.
I added the test in test_feature_store_bq_vectorstore.py and test are passing locally